src/service/tempMailApi.js

```diff
@@ -6,11 +6,24 @@ class TemporalMailService {
   constructor() {
     this.BASE_API_URL = 'https://api.mail.tm';
     this.httpClient = require('axios');
+    this.emailDomain = '';
   }
+
+  async _getDomains() {
+    try {
+      const response = await this.httpClient.get(`${this.BASE_API_URL}/domains`);
+      const {domain} = response.data['hydra:member'];
+      this.emailDomain = domain;
+    } catch (err) {
+      throw new Error('Cant get domains for emails');
+    }
+  }
+
   async createMail({mail, password}) {
     try {
+      await this._getDomains();
       const response = await this.httpClient.post(`${this.BASE_API_URL}/accounts`, {
-        address: `${mail}@karenkey.com`,
+        address: `${mail}@${this.emailDomain}`,
```

Comment:
Looks like you're creating a global value, and repeating it across the code... maybe this should be one property of the class?